s2core/migrate/mysql/schema.sql

```diff
@@ -80,6 +80,7 @@ CREATE TABLE `labels` (
   `hbase_table_ttl` integer,
   `schema_version` varchar(8) NOT NULL default 'v2',
   `is_async` tinyint(4) NOT NULL default '0',
+  `compressionAlgorithm` varchar(64) NOT NULL DEFAULT 'lz4',
```

Comment:
like we discussed, better default value should be 'gz'. I added hbase.table.compression.algorithm into s2graph-deploy for use different default algorithm per deploy-phase. wondering what you think.